src/backend/main/py.main/caliopen_main/user/core/new_user_setups.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
```

Comment:
name to setups.py not a new_user_setups.py

Reply:
=> commit #68c73ee